Rakefile

```diff
@@ -9,8 +9,7 @@ task default: :test
 desc "Run tests"
 task :test do
   Rake::TestTask.new do |t|
-    t.libs << "lib" << "spec"
```

Comment:
`libs` option already contains `"lib"`, and since `"spec"` dir is not used we can delete this line and just use the default `libs` value.